### ISSUE TITLE
When bottom toolbar is enabled, it is difficult to use Touch to Search

### ISSUE THREAD

**Issue by cbwee** [kiwibrowser/src.next#224]
**Describe the bug**
When bottom toolbar is enabled, it is difficult to use Touch to Search.

Because Touch to Search can't be opened when it's covered by the toolbar, so I have to scroll to hide the toolbar to open Touch to Search, please see the screen recording.

**To Reproduce**
1. Enable bottom toolbar in Kiwi Browser.
2. Enable Touch to Search.
3. Open any webpage.
4. Select any text on the webpage.
5. Try to open Touch to Search, it can't be opened.
6. Scroll down to hide the toolbar
7. Select the text again, now without the toolbar, Touch to Search can be opened.

**Expected behavior**
I should not have to hide the toolbar to be able to open Touch to Search, I think:
1. Touch to Search should be appeared above the toolbar
or 
2. The browser should hide the toolbar automatically when Touch to Search is displayed.

**Screen recording**

https://user-images.githubusercontent.com/34395942/127590356-5cb4274d-bd51-4ed2-9965-92c6dbe4b024.mp4

**Smartphone (please complete the following information):**
 - Device: Xiaomi POCO M3
 - Android Version : Android 10
 - Kiwi Browser Version 93.0.4577.18

**Comment by joelocandol** [comment]
This problem also happens to me 
Oukitel c5 Android 7.0

**Comment by kiwibrowser** [comment]
Fixed in last build

**Comment by Lartsch** [comment]
Yep, this also occured on Kiwi version before the latest rebase.
